### PR TITLE
added filter for loguru

### LINKED ISSUE #180: Identify and resolve source of warnings in `record.py`
```
tate:42 - exc=TypeError('Cannot pickle Objective-C objects')
2023-05-27 01:37:16.445 | WARNING  | openadapt.window._macos:get_active_window_state:42 - exc=TypeError('Cannot pickle Objective-C objects')
2023-05-27 01:37:16.546 | WARNING  | openadapt.window._macos:get_active_window_state:42 - exc=TypeError('Cannot pickle Objective-C objects')
2023-05-27 01:37:16.688 | WARNING  | openadapt.window._macos:get_active_window_state:42 - exc=TypeError('Cannot pickle Objective-C objects')
... 

@0dm thank you for reporting! Please post a more complete example 🙏

### PR DESCRIPTION
**Pull request by 0dm**
- resolves #180 using a filter for loguru 

# video
https://streamable.com/y5dk24 before/after

### CODE REVIEW

**Comment by 0dm** [comment]
regarding this:

**Review by abrichr — CHANGES_REQUESTED** [review]

File: openadapt/utils.py [hunk 1]
```diff
@@ -28,10 +28,18 @@ def configure_logging(logger, log_level):
...
+    logger.add(sys.stderr, level=log_level, filter=good_filter)
     logger.debug(f"{log_level=}")
 
 
+def good_filter(data):
+    messages_to_ignore = [
+        "Cannot pickle Objective-C objects')",
+    ]
+
+    return not any(data["message"].endswith(msg) for msg in messages_to_ignore)
```
[the remark this answers is not in the document]
Comment (0dm) [reply openadapt/utils.py]: sure, somehow didn't think of that

**Comment by 0dm** [comment]
@abrichr should be good now

File: openadapt/utils.py [hunk 2]
```diff
@@ -28,10 +28,17 @@ def configure_logging(logger, log_level):
     log_level_override = os.getenv("LOG_LEVEL")
     log_level = log_level_override or log_level
     logger.remove()
-    logger.add(sys.stderr, level=log_level)
+    logger.add(sys.stderr, level=log_level, filter=filter_log_messages)
     logger.debug(f"{log_level=}")
 
 
+def filter_log_messages(data):
+    messages_to_ignore = [
```
Comment (abrichr) [inline openadapt/utils.py:36]: How about adding if `config.FILTER_WARNINGS:` here?

**Pull request closed without merging**